wsmaster/che-core-api-workspace/src/test/java/org/eclipse/che/api/workspace/server/spi/tck/WorkspaceDaoTest.java

```diff
@@ -667,6 +704,119 @@ public static WorkspaceImpl createWorkspace(String id, AccountImpl account, Stri
     return workspace;
   }
 
+  public static WorkspaceImpl createWorkspaceFromDevfile(
```

Comment:
Please note that there is not any a case that does not require a Devfile specific test. Now I see that `create workspace`, `update workspace` is tested only with workspace config.

Reply:
right, added more tests